vscode-dotnet-runtime-library/src/Acquisition/IGlobalInstaller.ts

```diff
@@ -0,0 +1,32 @@
+/* --------------------------------------------------------------------------------------------
+ *  Licensed to the .NET Foundation under one or more agreements.
+*  The .NET Foundation licenses this file to you under the MIT license.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+
+import path = require('path');
+import { IAcquisitionWorkerContext } from './IAcquisitionWorkerContext';
+
+export abstract class IGlobalInstaller {
+
+    protected acquisitionContext : IAcquisitionWorkerContext;
+
+    constructor(acquisitionContext : IAcquisitionWorkerContext) {
+        this.acquisitionContext = acquisitionContext;
+    }
+
+    public abstract installSDK() : Promise<string>
+
+    public abstract getExpectedGlobalSDKPath(specificSDKVersionInstalled : string, installedArch : string) : Promise<string>
+
+    public abstract getGlobalSdkVersionsInstalledOnMachine() : Promise<Array<string>>;
+
+    /**
+     *
+     * @returns The folder where global sdk installers will be downloaded onto the disk.
+     */
+    public static getDownloadedInstallFilesFolder() : string
+    {
+        return path.join(__dirname, 'installers');
```

Comment:
Indeed, it would go into say c:\Users\USER\.vscode\extensions\ms-dotnettools.vscode-dotnet-sdk-2.0.0\dist\installers

However it deletes every installer file right after its done executing. We could try to go two folders up and dump it into the /extensions folder, but given it doesn't hang onto the installers, I'm not sure this is a big deal. We likely wouldn't need to use the installer more than once.